src/poliastro/constants/rotational_elements.py

```diff
@@ -0,0 +1,209 @@
+from astropy.constants import Constant
+
+T_earth = Constant(
+    "T_earth",
+    "Earth rotational period",
+    0.9972698,
+    "d",
```

Comment:
Shouldn't the unit be angles per unit time, like deg/second or similar?

Reply:
It would probably make more sense but this is how it was referenced so I kept it like this for consistency

Reply:
Well, if it's a period then it makes total sense that it's in [time] units only, so let's leave it like that.